### ISSUE TITLE
Add more countries

### ISSUE THREAD

**Issue by federicoalvarez-github** [home-assistant/home-assistant.github.io#36218]
### Feedback

I see only 4 countries when configuring the integration, is it possible to add more? In particular I'm interested in Uruguay. Please let me know. Thanks in advance, Federico.

### URL

https://www.home-assistant.io/integrations/prosegur/

### Version

2024.12.1

**Comment by dgomes** [comment]
This issue does not relate to home assistant documentation, and will be closed

But FYI: https://github.com/dgomes/pyprosegur/issues/4

**Comment by federicoalvarez-github** [comment]
> This issue does not relate to home assistant documentation, and will be closed
> 
> But FYI: [dgomes/pyprosegur#4](https://github.com/dgomes/pyprosegur/issues/4)

Hello DGomes! Thanks for taking the time to review and reply.

I have tested you very API with Uruguay services and it works to perfection. I did not have to modify anything.

Using your "installation.py" I was able to do everything.

IMHO, just using version 0.0.12 of "pyprosegur" from the Home Assistant integration will totally enable Uruguay as an option.

(I'm not so sure if that will add the option in the combo box when integrating, but your code 0.0.12 is fully compatible with UY services!)

Is it possible to review this?

Thanks in advance!!!
